conan/conanfile.py

```diff
@@ -0,0 +1,61 @@
+###############################################################################
+##  This file is part of the KDAB State Machine Editor Library.
+##
+##  Copyright (C) 2014-2019 Klarälvdalens Datakonsult AB, a KDAB Group company, info@kdab.com.
+##  All rights reserved.
+##
+##  Licensees holding valid commercial KDAB State Machine Editor Library
+##  licenses may use this file in accordance with the KDAB State Machine Editor
+##  Library License Agreement provided with the Software.
+##
+##  This file may be distributed and/or modified under the terms of the
+##  GNU Lesser General Public License version 2.1 as published by the
+##  Free Software Foundation and appearing in the file LICENSE.LGPL.txt included.
+##
+##  This file is provided AS IS with NO WARRANTY OF ANY KIND, INCLUDING THE
+##  WARRANTY OF DESIGN, MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE.
+##
+##  Contact info@kdab.com if any conditions of this licensing are not
+##  clear to you.
+###############################################################################
+
+from conans import ConanFile, CMake, tools
+
+
+class KDStateMachineEditorConan(ConanFile):
+    name = "KDStateMachineEditor"
+    version = "1.2"
+    license = "https://raw.githubusercontent.com/KDAB/KDStateMachineEditor/1.2/LICENSE.txt"
```

Comment:
LICENSE.txt is the commercial license.  This is a dual-licensed product and can have either a commercial license (LICENSE.txt) or an LGPL license (LICENSE.LGPL.txt).  Can we have both licenses mentioned somehow?